File: programme/area-of-rhombus/area-of-rhombus.cs
// Write a program to compute the area of a rhombus
using System;

public class AreaOfRhombus
{
    static void Main()
    {
        float d1, d2, area;
        Console.Write("Diagonal 1 : ");
        d1 = Single.Parse(Console.ReadLine());
        Console.Write("Diagonal 2 : ");
        d2 = Single.Parse(Console.ReadLine());
        area = d1 * d2 / 2;

        Console.WriteLine("\nArea : " + area);
    }
}

File: programme/area-of-rhombus/area-of-rhombus.php
<?php
// Write a programme to find the area of a rhombus

function calculate_area(int|float $diag_one, int|float $diag_two) {

    if (!is_numeric($diag_one)) {

        return 'Invalid input: Diagonal 1 is not a number';

    }

    if (!is_numeric($diag_two)) {

        return 'Invalid input: Diagonal 2 is not a number';

    }

    $area = ($diag_one * $diag_two) / 2;

    return $area;

}

$diag_one = readline("Enter Diagonal Value #1: ");
$diag_two = readline("Enter Diagonal Value #2: ");
$area = calculate_area($diag_one, $diag_two);

print "\nDiagonal #1: {$diag_one}";
print "\nDiagonal #2: {$diag_two}";
print "\nArea: {$area}";

?>